pyop2/base.py

```diff
@@ -464,35 +465,46 @@ def _uses_itspace(self):
         return self._is_mat or isinstance(self.idx, IterationIndex)
 
     @collective
-    def halo_exchange_begin(self, update_inc=False):
```

Comment:
Okay, so since we are on an `Arg`, we already know what the access descriptor is, so no need for `update_inc=...`.